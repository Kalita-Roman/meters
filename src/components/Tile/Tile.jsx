import React from 'react';
import './Tile.scss';

export default ({ name }) => (<div className="tile">
    <div className="tile-header">{name}</div>
    <div className="tile-body"></div>
    <div className="tile-footer"></div>
</div>);