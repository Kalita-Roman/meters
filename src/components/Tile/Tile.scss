.tile {
    width: 50px;
    margin: 5px;
    padding: 5px;
    border-radius: 3px;
    background-color: #ccc;
}

.tile-header {
    width: 100%;
    border-radius: 3px;
    margin-bottom: 5px;
    background-color: #aaa;
}

.tile-body {
    width: 100%;
    min-height: 20px;
    margin-bottom: 5px;
    background-color: #aaa;
}

.tile-footer {
    width: 100%;
    height: 10px;
    background-color: #aaa;
}