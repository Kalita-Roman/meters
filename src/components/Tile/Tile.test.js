import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Tile from './Tile';

test('Tile.jsx', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Tile/>);
    expect(result).toMatchSnapshot();
});