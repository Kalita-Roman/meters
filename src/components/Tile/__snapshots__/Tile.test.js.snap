// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile.jsx 1`] = `
<div
  className="tile"
>
  <div
    className="tile-header"
  />
  <div
    className="tile-body"
  />
  <div
    className="tile-footer"
  />
</div>
`;
